### PR TITLE
CM-222: Delete ThumbnailUrl

### PR DESCRIPTION
**Pull request by ayumi-oxd**
### Jira Ticket:
CM-222

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/CM-222

### Description:
- Deleted `thumbnailUrl` from `parkPhotos`
- Prioritized the prefixed image src with `small_` on `find a park` page and `quick view` component
- Displayed the default image src without `small_` if the prefixed one is not available

### CODE REVIEW

File: src/staging/src/utils/helpers.js [hunk 1]
```diff
@@ -30,3 +30,11 @@ export const renderBreadcrumbs = (menuContent, pageContext) => {
         return breadcrumbItems.reverse()
     }
 }
+export const addPrefix = (str) => {
```
Comment (molund) [inline src/staging/src/utils/helpers.js:33]: Since this is a generic helpers file, the method name should be more descriptive.  e.g. `addSmallImagePrefix`

**Comment by ayumi-oxd** [comment]
Added a fallback for the prefixed `imageUrl`

**Comment by foley-lynn** [comment]
Error checking needs to be improved. Try putting in an url to an image that doesn't exist, rebuild Gatsby, and keep an eye on the console when you try to load the image.

**Comment by molund** [comment]
@ayumi-oxd can this be manually merged into the Strapi 4 branch?

**Pull request closed without merging**